exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity  >= roomMaxCapacity ; // finish this statement
```

Comment:
The enoughSpaceInRoom variable should be true if there is enough space for both mentors and students. You may want to compare roomMaxCapacity to something else.